book/chapters/zhang/main.tex

```diff
@@ -0,0 +1,193 @@
+    \caption{Comparison between simulation and experimental results}
+    \label{fig:experiment}
+\end{figure}
+
+Except for mesh and time sensitivities analysis, the simulation results should validated against the experiment. As shown in Figure \ref{fig:experiment}. The general trend of case 1.2 million elements(4.7 million DOFs) and measurement data are the same, so we think the simulation accuracy is acceptable. However, there is still a large space to improve the accuracy. Such as introducing nonlinear material properties. These parameters, such as thermal conductivity, heat capacity, and coefficient of friction are all not constant or linear with temperature, velocity, and pressure. Getting the exact material characteristics is difficult. So better numerical results can benefit from the research of tribology and material engineering. The more detailed experiment parameters, like the loading pressure, would also significantly improve the numerical results since the experiment tests also contain large variances.
+
+
+
+\subsection*{Average and maximum temperatures}
+
+This section presents the temperatures of brake discs with different contact areas. The average and the maximum temperatures are presented.
+The total contact surface is 200 cm$^2$. The brake pressures from the back of the brake pads are the same, 0.274 MPa, while the different contact areas will affect the contact pressure between the brake pads and discs. 20\%, 50\% and 100\% contact areas are compared. 20 \% represent the research from Eriksson\cite{eriksson_nature_2002}, and the 100\% contact areas represent most FEM or analytical solutions.
+
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.8\textwidth]{book/chapters/zhang/graphics/T_ave_dc.pdf}
+    \caption{Average temperatures with different contact areas}
+    \label{fig:T_ave}
+\end{figure}
+
+As shown in Figure \ref{fig:T_ave} is the average temperature for these three cases. The maximum temperature difference is 20 $^{\circ}\text{C}$ between 20\% and 100\% contact areas. The maximum relative difference is 16.6\%. The average temperature is not sensitive to different contact areas since the total heat input is the same, while only heat dissipation is slightly different because the temperature distribution of the brake discs is uneven. Uneven temperature distribution can be proven through the comparison of the maximum temperatures.
+
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.8\textwidth]{book/chapters/zhang/graphics/T_max_dc.pdf}
+    \caption{The maximum temperatures with different contact areas}
+    \label{fig:T_max}
+\end{figure}
+
+As shown in Figure \ref{fig:T_max} is the maximum temperature of brake discs. The maximum temperature difference is more than 300 $^{\circ}\text{C}$, and the relative difference can reach 160\%. Small contact areas induce higher local temperatures since contact pressure is significantly increased and all friction heat is loaded on limited surfaces.
```

Comment:
```suggestion
Figure \ref{fig:T_max} shows the maximum temperature of the brake discs. The maximum temperature difference is more than 300 $^{\circ}\text{C}$, and the relative difference can reach 160\%. Small contact areas induce higher local temperatures since contact pressure is significantly increased and all friction heat is loaded on limited surfaces.
```